src/reducers/angler_reducers.js

```diff
@@ -9,8 +9,14 @@ export default (state=[], action) => {
                 fish.weight = Math.round(fish.girth*fish.girth*fish.length/800)
                 return fish
             });
-            const sortedFish = unsortedFish.filter(fish => fish.species === 'channel').sort((a, b) => b.weight - a.weight).slice(0,5)
-            return sortedFish;
+            const channelFish = unsortedFish.filter(fish => fish.species === 'channel').sort((a, b) => b.weight - a.weight).slice(0,5)
+            const flatFish = unsortedFish.filter(fish => fish.species === 'flat').sort((a, b) => b.weight - a.weight).slice(0,5);
+            const blueFish = unsortedFish.filter(fish => fish.species === 'blue').sort((a, b) => b.weight - a.weight).slice(0,5)
+            return {
```

Comment:
this aggregation is spurious. i'd expect you to use `.reduce`. this would violate DRY principles. If you wish to attempt this portion and are unfamiliar with the reduce function, you can reference https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce

Reply:
to help you out, the reduce signature is would look like:

```javascript
const aggregation = {};
const aggregatedFish = fish.reduce((previous, next) => { 
    // your work reducing
}, aggregation);
```

Typically I wont set a variable `aggregation` for this, I just want to be verbose. Typically you will see this online as:

```javascript
const aggregatedFish = fish.reduce((previous, next) => { 
    // your work reducing
}, {});
```